src/Resource/Otp.php

```diff
@@ -52,22 +52,58 @@ public function send(string $to, string $text, string $from = null, string $code
     }
 
     /**
-     * @param string $requestId
+     * Verify an OTP code entered by your customer using request ID received upon sending an OTP
+     *
+     * @param string $requestId Request ID
+     * @param string $code      OTP code
+     *
+     * @return array
+     * @throws AuthenticationException
+     * @throws InvalidPayloadException
+     * @throws ResourceNotFoundException
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public function verfiyByRequestId(string $requestId, string $code)
+    {
+        $uri = $this->prepareApiUri($this->resourceUri . '/requestid/' . $requestId . '/validate');
```

Comment:
Lets validate `$requestId` to make sure the user can't send any malicious content.